clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -285,6 +285,10 @@ Changes in existing checks
   <clang-tidy/checks/modernize/use-using>` check to fix function pointer and
   forward declared ``typedef`` correctly.
 
+- Improved :doc:`modernize-use-using
```

Comment:
merge with previous entry for this check.

Reply:
done